Addition of MEV Blocker docs to the docs. (#352)
cowprotocol/docs: docs/mevblocker/concepts/MEV Concepts/what-is-backrunning.md

---
sidebar_position: 7
---

> # What is Backrunning?

Reviewer: ```suggestion
---

# What is Backrunning?
```

Newline

Backrunning is a specific type of MEV (Maximal Extractable Value) which causes traders to miss out on potential profit from their trades.

In practice, backrunning involves strategically executing a transaction immediately after another, high-value transaction. By doing this, the backrunning transaction capitalizes on the arbitrage opportunity left over from the [price impact](https://blog.cow.fi/cow-protocol-glossary-ad7313a2c71#:~:text=price%20impact) of the initial transaction.

![backrunning](/img/mevblocker/backrunning.webp)

> Backrunning involves the tactical use of the information contained within the Ethereum mempool — a holding area for all pending transactions — to place transactions immediately after high-value trades in order to profit.

Reviewer: ```suggestion
Backrunning involves the tactical use of the information contained within the Ethereum mempool - a holding area for all pending transactions - to place transactions immediately after high-value trades in order to profit.
```

No em-dash. Use hyphen.

> Out of the three types of MEV (frontrunning, sandwich attacks, and backrunning) backrunning is considered the least harmful. In fact, if performed by itself, this method simply captures the arbitrage opportunity left over by a large trade without affecting the initial trade itself. Backrunning is only harmful to the user when combined with a frontrunning attack, which results in the worst kind of MEV — the sandwich attack.

Reviewer: ```suggestion
Out of the three types of MEV (frontrunning, sandwich attacks, and backrunning) backrunning is considered the least harmful. In fact, if performed by itself, this method simply captures the arbitrage opportunity left over by a large trade without affecting the initial trade itself. Backrunning is only harmful to the user when combined with a frontrunning attack, which results in the worst kind of MEV - the sandwich attack.
```

### Backrunning example

To better understand the concept of backrunning, let's consider an example scenario. In this example, a user named Alex initiates a substantial trade, selling $10,000 of ETH to buy COW on Uniswap. Due to the nature of AMMs, this swap leads to a jump in the value of COW and a drop in the value of ETH on this particular exchange (Uniswap).

At the same time, an MEV bot notices Alex's large transaction and springs into action. The MEV bot acquires Alex's sell token (ETH) at this lower price and immediately turns around and sells it for a higher price on a different exchange where this price arbitrage has not yet been reflected. The bot profits from the price difference between the assets on these two exchanges.

> From a technical perspective, Alex's transaction doesn’t result in any realized loss for him, since he still got the price he was quoted for his trade. However, his trade created an arbitrage opportunity that he didn’t capture, leaving money on the table.

Reviewer: ```suggestion
From a technical perspective, Alex's transaction doesn't result in any realized loss for him, since he still got the price he was quoted for his trade. However, his trade created an arbitrage opportunity that he didn't capture, leaving money on the table.
```

Single typewriter apostrophe :pray:
